Sources/Commands/Options.swift

```diff
@@ -348,6 +348,9 @@ public struct SwiftToolOptions: ParsableArguments {
           help: .hidden)
     var keychain: Bool = false
 #endif
+    
+    @Flag(help: "Fingerprint check failures would result in errors")
+    var strictFingerprintChecking: Bool = false
```

Comment:
maybe we make this into a `fingerprintCheckingMode` enum with "none", "warn" and "strict" options, so in the future we can change the default but users can still choose a different mode

Reply:
https://github.com/apple/swift-package-manager/pull/3890/commits/578ba66a5f4a6e4103b9a8e82cbd73c6d212457b